raspi/controlserver/controlserver.py

```diff
@@ -110,6 +123,19 @@ def mac_id():
     resp = jsonify(dm)
     return resp
 
+@app.route('/unlink_device', methods=['GET', 'POST'])
+def unlink_device():
+    mac = return_mac()
+    access_token = get_token()
+    PARAMS = {
+        'macid':mac,
```

Comment:
Please care the whitespace.

Reply:
@hongquan I don't see any trailing whitespace. which whitespace needs to be removed/added? 
should it be `'macid': mac` ? sorry for the amateur question!

Reply:
Right.

Reply:
updated